bin/make_view

```diff
@@ -43,24 +45,28 @@ CREATE OR REPLACE VIEW "Measure_Expenditures" AS
   SELECT
     expend."Filer_ID"::varchar,
     expend."Filer_NamL",
+    "election_name",
     "Bal_Name",
     "Ballot_Measure" as "Measure_Number",
     "Support_Or_Oppose" as "Sup_Opp_Cd",
     "Amount",
     "Payee_NamL" as "Recipient_Or_Description"
   FROM
     "E-Expenditure" expend,
-    oakland_committees committee
+    oakland_committees committee,
+    oakland_name_to_number
```

Comment:
Can we trend towards using JOINs when selecting from multiple tables? I find it a lot easier to read the intent of the query that way since the join condition is right next to the table name. It's equivalent, right?